### DISCUSSION TITLE
Yolov8 Metrics' Plot Properties

### DISCUSSION THREAD

**Discussion by TRDWuisan** [ultralytics/ultralytics#1916] [Q&A]
Heya folks, a bit of a basic question here (i hope), but I'm using roboflow's yolov8 tutorial and am wondering how to change the formatting of the metric plots (f1, precision, recall curves, & confusion matrix). or better yet, how can I call the values/variables that are used to plot these curves as in their tutorial the plots are saved as pngs directly. I know there's the results.csv file that includes the values for the training/validation/mAP, but it doesnt seem to include the confusion matrix/precision/recall curves. I just want to change the formatting of the plots like turning on latex. Any ideas? Thanks :D

**Comment by glenn-jocher** [comment]
@TRDWuisan you can result validation results directly in Python and handle according to your custom needs there:

```python
from ultralytics import YOLO

model = YOLO('yolov8n.pt')
results = model.val(data='coco128.yaml')
```

For details see https://docs.ultralytics.com/modes/val/

<img width="729" alt="Screenshot 2023-04-12 at 01 51 34" src="https://user-images.githubusercontent.com/26833433/231312640-7df2398b-8717-4b0d-b3a5-525ab2e64f0a.png">
